toolkits/global/packages/global-button/README.md

```diff
@@ -29,9 +29,25 @@ Button styles to use on buttons and links.
 <button class="c-button c-button--primary">text</button>
 ```
 
-**Block**
+**Full width**
```

Comment:
Any reason for this one?

Reply:
I thought it might be potentially confusing to call it block when it has `display: flex` on it.

Also changed this to flex so that it can be used with the icon aligning classes. So that you can use the icon modifier without it being full width.

Reply:
👍